optuna/study.py

```diff
@@ -472,14 +473,34 @@ def create_study(
         direction:
             Direction of optimization. Set ``minimize`` for minimization and ``maximize`` for
             maximization. Note that ``maximize`` is currently unsupported.
+        exist_ok:
+            Flag to control the behavior to handle a conflict of study names.
+            In the case where a study named ``study_name`` already exists in the ``storage``,
+            a :class:`~optuna.structs.DuplicatedStudyError` is raised if ``exist_ok`` is
+            set to :obj:`False`.
+            Otherwise the creation of the study is skipped and no exception is raised.
 
     Returns:
         A :class:`~optuna.study.Study` object.
 
     """
 
     storage = storages.get_storage(storage)
-    study_name = storage.get_study_name_from_id(storage.create_new_study_id(study_name))
+    try:
+        study_id = storage.create_new_study_id(study_name)
+    except structs.DuplicatedStudyError:
+        if exist_ok:
+            assert study_name is not None
+
+            logger = logging.get_logger(__name__)
+            logger.info('The study name {} is duplicated with the existing study.'
+                        'It is reused instead of creating a new one.'.format(study_name))
```

Comment:
How about something like :`Using an existing study with name '{}' instead of creating a new one` or `Connecting to an existing study with name '{}' instead of creating a new one` ?